projects/kit/src/lib/time/constants/default-time-segment-max-values.ts

```diff
@@ -0,0 +1,8 @@
+import {MaskitoTimeSegments} from '../types';
+
+export const DEFAULT_TIME_SEGMENT_MAX_VALUES: MaskitoTimeSegments<number> = {
+    hours: 23,
```

Comment:
Just something to keep in mind, we'd want to support AM/PM mode eventually too.

Reply:
https://github.com/Tinkoff/maskito/pull/37#discussion_r1046913776